src/db/config/config.js

```diff
@@ -4,7 +4,7 @@ const config = {
   development: {
     username: envConfig.db_username,
     password: envConfig.db_password,
-    database: 'authorsHavenDb',
+    database: 'authorshavendb',
```

Comment:
what's the reason for changing the db name?

Reply:
sorry for that I somehow forgot to change it back while testing something else on my local machine due to the previous rebase